### PR TITLE
app-editors/gnome-text-editor: Version bump to 43.0

### PR DESCRIPTION
**Pull request by Schievel1**

### CODE REVIEW

File: app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="A simple text editor for the GNOME desktop"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-text-editor"
+SRC_URI="mirror://gnome/sources/gnome-text-editor/42/gnome-text-editor-${PV/_/.}.tar.xz"
```
Comment (mattst88) [inline app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild:10]: Just drop `SRC_URI=...`. It's wrong, and I'm not sure why it's there in the v42 ebuild.

````
>>> Downloading 'https://download.gnome.org/sources/gnome-text-editor/42/gnome-text-editor-43.0.tar.xz'
--2022-09-21 09:00:00--  https://download.gnome.org/sources/gnome-text-editor/42/gnome-text-editor-43.0.tar.xz
Resolving download.gnome.org (download.gnome.org)... 8.43.85.3, 8.43.85.4, 8.43.85.5
Connecting to download.gnome.org (download.gnome.org)|8.43.85.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-09-21 09:00:00 ERROR 404: Not Found.

!!! Couldn't download 'gnome-text-editor-43.0.tar.xz'. Aborting.
````

File: app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	>=dev-libs/glib-2.73.0:2
+	>=gui-libs/gtk-4.7:4
+	>=gui-libs/gtksourceview-5.5.0:5
+	>=gui-libs/libadwaita-1.2:1
+	dev-libs/libpcre
+	spell? (
+		>=app-text/enchant-2.2.0:2
+		dev-libs/icu:=
+	)
+	app-text/editorconfig-core-c
```
Comment (mattst88) [inline app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild:30]: This should go right after libadwaita:

````diff
-libadwaita_dep = dependency('libadwaita-1')
+libadwaita_dep = dependency('libadwaita-1', version: '>= 1.2.alpha')
+libeditorconfig_dep = dependency('editorconfig')
````

**Review by mattst88 — CHANGES_REQUESTED** [review]

**Comment by mattst88** [comment]
Modulo the two things, this looks good. I'll fix those locally and push it. Thanks!

**Comment by mattst88** [comment]
FWIW, the libpcre dep was removed in upstream commit

````
commit 6ea5bbae19acdcce45044ad91b7b28f8a7e5870d
Author: Christian Hergert <chergert@redhat.com>
Date:   Fri Jul 22 10:24:24 2022 -0700

    build: switch to libeditorconfig
    
    We don't need our vendored libeditorconfig anymore now that the upstream
    version is more widely available. More importantly, it's moved on to
    PCRE2 and we want to drop PCRE1 altogether now that GLib has too.
````

File: app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild (hunk 1 above)
Comment (Schievel1) [reply app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild:10]: I couldn't get it to work without the SRC_URI, I figured that upstream does something with where they put the file that is different from what the other gnome people do.

**Pull request closed without merging**